src/main/java/edu/harvard/iq/dataverse/util/FileUtil.java

```diff
@@ -545,6 +562,11 @@ public static String determineFileType(File f, String fileName) throws IOExcepti
     }
 
     public static String determineFileTypeByNameAndExtension(String fileName) {
+        final ResourceBundle bundle = BundleUtil.getResourceBundle("MimeTypeDetectionByFileName");
```

Comment:
It looks like this obsoletes some of lookupFileTypeFromPropertiesFile - that ends up trying to check the same file again after checking based on extension. Can that logic be removed/otherwise refactored to just prioritize the check by full name?